ios/MaasCore/Sources/MaasCore/Api/ApiError.swift

```diff
@@ -19,17 +19,26 @@ public enum ApiError: Swift.Error, Equatable, Hashable {
      - Parameter error: optional error message.
      */
     case error(error: CoreBinary.Error?)
+    
+    case errorCode(code: Code, error: CoreBinary.Error?)
+    
```

Comment:
Let's discard `case error(error: CoreBinary.Error?)` and use `case error(code: Code, error: CoreBinary.Error?)` instead.

Reply:
but code will not always be there 🤔 I would prefer separate enum case instead of optional `Code` but there is room for improvement 🤔

Reply:
Lets make it optional then 😄